### ISSUE TITLE
Installing UCell for R-4.1 and Bioconductor < 3.15

### ISSUE THREAD

**Issue by QichenFU** [carmonalab/UCell#24]
<img width="1026" alt="Screen Shot 2023-01-25 at 2 42 47 PM" src="https://user-images.githubusercontent.com/61186744/214500071-4aa34029-06aa-4d9a-889d-1715a9dab9b2.png">

I am trying to install UCell, but got above error. May I ask the solution?

**Comment by mass-a** [comment]
Hello,

you are trying to install on R-4.1 and Bioconductor 3.14 - UCell is only available for Bioconductor 3.15 or higher.

You should still be able to install previous versions from GitHub, by specifying the version:
```r
library(remotes)
remotes::install_github("carmonalab/UCell", ref="v1.3")
```

-m

**Comment by QichenFU** [comment]
It works. Thank you very much!
